cmd/helm/list.go

```diff
@@ -46,8 +47,8 @@ regular expressions (Perl compatible) that are applied to the list of releases.
 Only items that match the filter will be returned.
 
     $ helm list --filter 'ara[a-z]+'
-    NAME                UPDATED                     CHART
-    maudlin-arachnid    Mon May  9 16:07:08 2016    alpine-0.1.0
```

Comment:
I think this example needs to be reverted, if the default remains "Mon May  9 16:07:08 2016"

Reply:
I guess this example was copied over during migration from helm2. Actual default now is `2020-06-18 14:17:46.125134977 +0000 UTC`.